src/recommendations/testing.md

```diff
@@ -0,0 +1,44 @@
+---
+group: product-recommendations
+title: Verifying Recommendations
+---
+
+After you [install]({{ page.baseurl }}/recommendations/install.html) and [configure]({{ page.baseurl }}/recommendations/configure.html) the product recommendations module, you can verify that the behavioral data is being sent to Magento. You can use developer tools available in Chrome or you can install the Snowplow Chrome extension.
+
+## Verify Using Developer Tools in Chrome
+
+1. To ensure that the `DataServices.js` file is loading on all site pages:
```

Comment:
Consider using the `{:.procedure}` MD attribute instead of a numbered list here and below. It will format the "To ensure..." text to a pseudo H4 heading. Then you can use a numbered list for the bulleted list the follows.